### PR TITLE
hotfix/getUserLastOpened

### PR DESCRIPTION
**Pull request by JaredJRoss**
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Documentation updated
- [ ] Unit tests added/updated
- [ ] Smoke testing relevant to authentication needed
- [ ] Clones involved and accounted for
- [ ] RDS migrations or other data manipulation necessary
- [ ] Dev tools or other infrastructure changed

### CODE REVIEW

File: backend/node_app/controllers/appStatsController.js [hunk 1]
```diff
@@ -544,7 +542,7 @@ class AppStatsController {
 				database: this.constants.MATOMO_DB_CONFIG.database
 			});
 			connection.connect();
-			const results = await this.queryPDFOpenedByUserId('steve', clone_name, connection);
+			const results = await this.queryPDFOpenedByUserId(['steve'], clone_name, connection);
```
Comment (JLyons1985) [inline backend/node_app/controllers/appStatsController.js:545]: why is 'steve' hardcoded?

**Review by JLyons1985 — CHANGES_REQUESTED** [review]
I don't think steve should be in there. Might be meaning to pass the userId instead?

File: backend/node_app/controllers/appStatsController.js (hunk 1 above)
Comment (JaredJRoss) [reply backend/node_app/controllers/appStatsController.js]: not sure removed it to use the userid now

**Review by JLyons1985 — DISMISSED** [review]

**Review by britton-jason — CHANGES_REQUESTED** [review]
Backend error on homepage

```gamechanger-web-node   | 2022-03-28T19:26:01.637Z error - - Uncaught exception has occurred Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')
gamechanger-web-node   |                                order by
gamechanger-web-node   |                                        documenttime desc
gamechanger-web-node   |                                limit 10' at line 5
gamechanger-web-node   | 2022-03-28T19:26:01.637Z error - - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')
gamechanger-web-node   |                                order by
gamechanger-web-node   |                                        documenttime desc
gamechanger-web-node   |                                limit 10' at line 5
gamechanger-web-node   |     at Query.Sequence._packetToError (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
gamechanger-web-node   |     at Query.ErrorPacket (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
gamechanger-web-node   |     at Protocol._parsePacket (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:291:23)
gamechanger-web-node   |     at Parser._parsePacket (/usr/src/app/node_modules/mysql/lib/protocol/Parser.js:433:10)
gamechanger-web-node   |     at Parser.write (/usr/src/app/node_modules/mysql/lib/protocol/Parser.js:43:10)
gamechanger-web-node   |     at Protocol.write (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:38:16)
gamechanger-web-node   |     at Socket.<anonymous> (/usr/src/app/node_modules/mysql/lib/Connection.js:88:28)
gamechanger-web-node   |     at Socket.<anonymous> (/usr/src/app/node_modules/mysql/lib/Connection.js:526:10)
gamechanger-web-node   |     at Socket.emit (events.js:400:28)
gamechanger-web-node   |     at Socket.emit (domain.js:475:12)
gamechanger-web-node   |     at addChunk (internal/streams/readable.js:293:12)
gamechanger-web-node   |     at readableAddChunk (internal/streams/readable.js:267:9)
gamechanger-web-node   |     at Socket.Readable.push (internal/streams/readable.js:206:10)
gamechanger-web-node   |     at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
gamechanger-web-node   |     --------------------
gamechanger-web-node   |     at Protocol._enqueue (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:144:48)
gamechanger-web-node   |     at Connection.query (/usr/src/app/node_modules/mysql/lib/Connection.js:198:25)
gamechanger-web-node   |     at /usr/src/app/node_app/controllers/appStatsController.js:258:15
gamechanger-web-node   |     at new Promise (<anonymous>)
gamechanger-web-node   |     at AppStatsController.queryPDFOpenedByUserId (/usr/src/app/node_app/controllers/appStatsController.js:256:10)
gamechanger-web-node   |     at AppStatsController.getUserLastOpened (/usr/src/app/node_app/controllers/appStatsController.js:899:31)
gamechanger-web-node   |     at processTicksAndRejections (internal/process/task_queues.js:95:5)
gamechanger-web-node   |     at async UserController.getUserData (/usr/src/app/node_app/controllers/userController.js:384:24)```

**Review by rha930 — APPROVED** [review]

**Review by britton-jason — APPROVED** [review]

**Pull request merged**